modules/meta/arrow-meta-prototype/idea-plugin/src/main/kotlin/arrow/meta/plugin/idea/plugins/nothing/NothingIdePlugin.kt

```diff
@@ -0,0 +1,18 @@
+package arrow.meta.plugin.idea.plugins.nothing
+
+import arrow.meta.Plugin
+import arrow.meta.invoke
+import arrow.meta.plugin.idea.IdeMetaPlugin
+import arrow.meta.plugin.idea.resources.ArrowIcons
+import org.jetbrains.kotlin.psi.KtUserType
+import org.jetbrains.kotlin.utils.addToStdlib.safeAs
+
+val IdeMetaPlugin.nothingIdePlugin: Plugin
+  get() = "NothingIdePlugin" {
+    meta(
+      addLineMarkerProvider(
+        icon = ArrowIcons.NOTHING_LIGHT,
+        message = "Nothing is impure"
```

Comment:
What is the purpose of giving a message that nothing is impure as a plugin?

Reply:
`Nothing` refers to the Type :)

Reply:
> "Nothing is impure"
That is a confusing message because "Nothing" doesn't implicate impureness perse but rather it's a [bottom type](https://en.wikipedia.org/wiki/Bottom_type).

For example `None` is `Option<Nothing>` but that doesn't have anything to do with purity.

Reply:
@raulraja @pakoito What should we use as a static message:
- `Bottom Type`
- `Nothing`

Or something else :)